packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -129,17 +127,19 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
     }, [amountInputName, composeTransaction, dispatch, inputName, setValue, symbol]);
 
     const getValidationButtonProps = ():
-        | { url: AddressDeprecatedUrl }
+        | { url: (typeof URLS)[NonNullable<ReturnType<typeof isAddressDeprecated>>] }
```

Comment:
nit: but I suggest t export the type and make it just one object (remove the unnecessary union `|`)

Reply:
ea22b6a95f4d1a2f63cc7e0f44918313274bb183